### PR TITLE
Feature/settings

### PR DESCRIPTION
**Pull request by cvializ**
@andrewconner ( @camhashemi ) remove the Kifi-admin experiment guard on team-settings. The incomplete settings tabs (like Plan Settings, billing, etc) are still Kifi-admin only. Add the Angular 404 for non-admins trying to access the team settings page.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js [hunk 1]
```diff
@@ -166,7 +174,7 @@ angular.module('kifi')
       return message;
     }
 
-    if ($scope.viewer.membership.role === 'admin') {
+    if ($scope.viewer && $scope.viewer.membership && $scope.viewer.membership.role === 'admin') {
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js]: nice catch. is `scope.viewer` ever not there?

**Comment by camhashemi** [comment]
:+1: I believe the 404 was the biggest hole in the whole thing, so we should be ready.

**Comment by camhashemi** [comment]
If we really wanted to be safe, we would deploy the new 404, test, then release the experiment. But it'd be rare to have `kifi.com/kifi/settings` without being a part of the org.

File: kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js]: yeah for some reason it wasn't there when I wasn't a member of the org

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/teamSettings.js]: that's bad, but good to know! I'll test it out

**Pull request merged**